src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -60,10 +62,10 @@ public KeyInfoX509Data(X509Certificate cert, X509IncludeOption includeOption)
                     // Can't honor the option if we only have a partial chain.
                     if ((chain.ChainStatus.Length > 0) &&
                         ((chain.ChainStatus[0].Status & X509ChainStatusFlags.PartialChain) == X509ChainStatusFlags.PartialChain))
-                        throw new CryptographicException(CAPI.CERT_E_CHAINING);
+                    throw new CryptographicException(SR.Cryptography_Partial_Chain);
```

Comment:
Ugh, yes - sorry, I'm still trying to get my VS editor settings to match what you guys use (agree that formatting makes no sense). I'll sort this out.